corgi/tasks/sca.py

```diff
@@ -156,28 +156,8 @@ def cpu_software_composition_analysis(build_uuid, force_process: bool = False):
     logger.info("Started software composition analysis for %s", build_uuid)
     software_build = SoftwareBuild.objects.get(pk=build_uuid)
 
-    component_qs = Component.objects.filter(software_build=software_build)
-    try:
-        # Get root component for this build.
-        root_component = component_qs.root_components().exclude(type=Component.Type.RPMMOD).get()
-    except Component.DoesNotExist as exc:
-        # None of the components were root components
-        module_qs = component_qs.filter(type=Component.Type.RPMMOD)
-        if len(module_qs) != 1:
-            logger.error(f"Build {build_uuid} had wrong number of modules: {len(module_qs)}")
-            # We have more than one module, or don't have any modules at all
-            # so we don't know which component / don't have any component to do SCA on
-            # we only do SCA on root components, so just fail the task
-            raise exc
-
-        # Else we have exactly one module component
-        # which is no longer considered a "root" component
-        # so now we skip doing SCA on the module instead of failing
-        logger.info(
-            f"Build {build_uuid} had only one module, no other root components. Skipping SCA"
-        )
-        return
-
+    # Get root component for this build.
+    root_component = Component.objects.filter(software_build=software_build).root_components().get()
```

Comment:
I think we should continue to skip RPMMOD when doing SCA. There is no real source code as far as I know, instead RPMMOD reuses content from another SRPM. We scan all the included RPMs separately when processing their respective SRPMs.